charts/kubernetes-external-secrets/values.yaml

```diff
@@ -52,7 +52,7 @@ replicaCount: 1
 
 image:
   repository: godaddy/kubernetes-external-secrets
-  tag: 3.2.0
+  tag: 3.2.1
```

Comment:
```suggestion
  tag: 3.2.0
```

Reply:
Reverted the version back to 3.2.0 and did a rebase.
Yes, it is a breaking change, thus I thought it might be better to do a version upgrade.
Let me know if anything else is needed.